src/main/java/core/basesyntax/service/impl/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.OperationHendler;
+import core.basesyntax.service.ReportGenerator;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    private static final String SEPARATOR = ",";
+    private FruitDao fruitDao;
+    private Map<FruitTransaction.Operation, OperationHendler> operationHendlerMap;
+
+    public ReportGeneratorImpl(FruitDao fruitDao,
+                               Map<FruitTransaction.Operation,
+                                      OperationHendler> operationHendlerMap) {
+        this.fruitDao = fruitDao;
+        this.operationHendlerMap = operationHendlerMap;
+    }
+
+    @Override
+    public String reportGenerater() {
```

Comment:
```suggestion
    public String generateReport() {
```

названия методов должны начинаться с глаголов